recipes/logr/0.1.0/conanfile.py

```diff
@@ -0,0 +1,88 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class LogrConan(ConanFile):
+    name = "logr"
+    version = "0.1.0"
+    license = "BSD 3-Clause License"
+    homepage = "https://github.com/ngrodzitski/logr"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Logger frontend substitution for spdlog, glog, etc for server/desktop applications"
+    topics = ("logger", "development", "util", "utils")
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = ["CMakeLists.txt"]
+
+    options = { "spdlog_backend" : [True, False],
+                "glog_backend" : [True, False],
+                "log4cplus_backend" : [True, False] }
+
+    default_options = { "spdlog_backend": True,
+                        "glog_backend": True,
+                        "log4cplus_backend": True }
```

Comment:
seem's a little odd to have 3 logging method's 🤔

Reply:
The relevant comment to similar comment
https://github.com/conan-io/conan-center-index/pull/3812#discussion_r537115162